satellite/rewards/rewards_test.go

```diff
@@ -59,27 +59,26 @@ func TestOffer_Database(t *testing.T) {
 			require.NoError(t, err)
 			require.Equal(t, new, c)
 
-			update := &rewards.UpdateOffer{
-				ID:        new.ID,
-				Status:    rewards.Done,
-				ExpiresAt: time.Now(),
-			}
+			//TODO: delete this test after removing Redeem method
 
-			isDefault := update.Status == rewards.Default
-			err = planet.Satellites[0].DB.Rewards().Redeem(ctx, update.ID, isDefault)
-			require.NoError(t, err)
+			//isDefault := new.Status == rewards.Default
+			//err = planet.Satellites[0].DB.Rewards().Redeem(ctx, new.ID, isDefault)
+			//require.NoError(t, err)
+			//
+			//current, err := planet.Satellites[0].DB.Rewards().ListAll(ctx)
+			//require.NoError(t, err)
+			//if current[i].Status == rewards.Default {
+			//	require.Equal(t, new.NumRedeemed, current[i].NumRedeemed)
+			//} else {
+			//	require.Equal(t, new.NumRedeemed+1, current[i].NumRedeemed)
+			//}
 
-			err = planet.Satellites[0].DB.Rewards().Finish(ctx, update.ID)
+			err = planet.Satellites[0].DB.Rewards().Finish(ctx, all[i].ID)
```

Comment:
is it possible for `i` to be greater than the length of `all` ?

Reply:
Looks like no- we insert one offer in each iteration, so the length of "all offers" will be at least i+1